src/sidebar/store/modules/selection.js

```diff
@@ -204,14 +203,18 @@ const update = {
     return setTab(state.selectedTab, action.tab);
   },
 
+  /**
+   * Automatically select the Page Notes tab, for convenience, if all of the
+   * top-level annotations in `action.annotations` are Page Notes and the
+   * previous annotation count was 0 (i.e. collection empty).
+   */
   ADD_ANNOTATIONS(state, action) {
```

Comment:
For the record, I think the current implementation of this feature—auto-selection of a tab depending on annotation collection contents—is a little naive and rickety. At some point in the future, I think we should re-evaluate the behavior (i.e. what happens in different combinations of annotation types? what if orphans are in the mix, e.g.?) and the implementation.

Reply:
Good question. Note that newly added annotations are never orphans. Rather an orphan is something that an annotation _temporarily becomes_ if it fails to be located in the page.